### DISCUSSION TITLE
Come settare campi per DDT/delivery_note

### DISCUSSION THREAD

**Discussion by cleoni** [fattureincloud/api#44] [Q&A] [answered]
Salve, sto usando APIv2 per trasmettere un DDT con PHP e a livello di prime informazioni funziona.

Devo aggiungere i seguenti campi (trasmessi senza problemi con APIv1) ma non trovo dove sono documentati i metodi da chiamare, visto che non posso passare i dati grezzi come coppia chiave/valore.

dn_ai_destination (luogo di destinazione)
dn_ai_causal (causale del trasporto) 
dn_ai_packages_number (numero di colli)
dn_ai_transporter (trasporto) 
dn_ai_notes (annotazioni)

Sono documentati?

**Comment by emajo** [comment, marked as answer]
Salve @cleoni, i campi da lei indicati sono documentati nella chiamata di creazione dei documenti emessi [qui](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/issued_documents).

**Reply by cleoni** [reply]
Nella pagina linkata c'è una lista di campi e una sintetica indicazione di cosa si riferiscono. Questo va bene, ma il codice indicato a esempio è una struttura JSON che, secondo quanto finora emerso, non si può usare direttamente.
C'è una API PHP da utilizzare che ha un singolo metodo da chiamare per settare ogni singolo campo, ed è proprio di quelli che non trovo documentazione. 

Comunque non importa, ho guardato il sorgente della libreria API e per il momento ho già risolto. Grazie comunque per la risposta,
